src/main/java/de/dhbw/karlsruhe/students/mailflow/core/application/auth/LoginService.java

```diff
@@ -46,7 +46,7 @@ private void authorizeUser(Address address, String password)
 
     var user = foundUser.get();
 
-    if (!passwordChecker.checkPassword(password, user)) {
+    if (!passwordChecker.checkPassword(password.trim(), user)) {
```

Comment:
not sure if we should do it here. I would prefer if you do it in the baseCLIPrompt so that it cleanses every userinput or we could introduce an anti-corruption layer somewhere. I'm not sure how it would look like with the ACL

Reply:
I added the functionality to simplePrompt() in the BaseCLIPrompt